swift/Arrow/Tests/ArrowTests/CodableTests.swift

```diff
@@ -18,7 +18,7 @@
 import XCTest
 @testable import Arrow
 
-final class CodableTests: XCTestCase {
+final class CodableTests: XCTestCase { // swiftlint:disable:this type_body_length
```

Comment:
Do we need this?
It seems that `testArrowKeyedEncoder` has this.

Reply:
The swiftlint rule that is disabled on testArrowKeyedEncoder is for the function length.  CodableTests needs to be refactored into two different test classes: one for Encoding and another for Decoding. (I plan to do this in a follow up PR)